staging/src/k8s.io/cri-api/pkg/apis/runtime/v1alpha2/api.proto

```diff
@@ -658,7 +657,16 @@ message LinuxPodSandboxStats {
 
 // WindowsPodSandboxStats provides the resource usage statistics for a pod sandbox on windows
 message WindowsPodSandboxStats {
-    // TODO: Add stats relevant to windows.
+    // CPU usage gathered for the pod sandbox.
```

Comment:
how close are we to dropping v1alpha2 @mrunalp @mikebrow ? I feel like we don't need to add this here

Reply:
I dropped the v1alpha2 for now.  Let me know if I should add it back

Reply:
Looks like there are tests that validate v1alpha2 https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/110754/pull-kubernetes-unit/1583552443948470272